roles/postgres/templates/postgres.deployment.yaml

```diff
@@ -22,12 +22,12 @@ spec:
             - name: POSTGRESQL_DATABASE
               value: pulp
             - name: POSTGRESQL_USER
-              value: "{{ database_connection.username }}"
+              value: "{{ database_settings.username }}"
```

Comment:
Aren't these duplicate of pulp_default_settings? They're not even defined in playbook.yaml anymore.

Reply:
We should keep the settings for the postgres container separate from the settings for Pulp. I am going to introduce 'database_settings' that will contain the same settings as 'database_connection' had before. These will only be used for configuring postgres.

Reply:
Can we merge (including renaming) some specific vars though? Otherwise, users must specify these twice.

Reply:
I don't think we should merge. Even though it seems like they user is entering settings twice, they are entering different settings. One creates an account in postgres. The other configures Pulp to use a specific account.

Reply:
I think in the future it should be possible to use the operator without the postgres container at all. Or with a lot more settings for postgres.